helpers/aws/functions.js

```diff
@@ -3,6 +3,10 @@ var regions = require(__dirname + '/regions');
 var AWS = require('aws-sdk');
 var helpers = require('../shared.js');
 
+function isValid(value) {
+    return (value !== undefined && value !== '' && value !== null);
+}
```

Comment:
this is unnecessary. you can just check (value && value.length) which will cover all these

Reply:
You are right. I have removed this function